api-gateway/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM java:8
+ADD api-gateway.jar app.jar
```

Comment:
You can use `filtering` configuration from plugin `docker-maven-plugin` to be able to 

```
ADD @project.build.finalName@.jar app.jar
```